### ISSUE TITLE
Method not found: 'Boolean ExpectoConfig.get_allowDuplicateNames()

### ISSUE THREAD

**Issue by adam-c-anderson** [adamchester/expecto-adapter#39]
Tests are discovered, but an unexpected error occurs when trying to run them.
The tests run fine from the command line.

Adapter was working fairly recently, but started doing this instead after I added some new tests. Removing the tests again did not restore test-running behavior.

```
[2/14/2018 11:37:19 AM Informational] ------ Load Playlist started ------
[2/14/2018 11:37:19 AM Informational] ========== Load Playlist finished (0:00:00.0110259) ==========
[2/14/2018 11:37:20 AM Informational] ------ Discover test started ------
[2/14/2018 11:37:23 AM Informational] ========== Discover test finished: 2 found (0:00:02.9418203) ==========
[2/14/2018 11:37:31 AM Informational] ------ Run test started ------
[2/14/2018 11:37:32 AM Informational] Running all tests (9999.0.0.0)
[2/14/2018 11:37:32 AM Informational] Creating test host for C:\Code\pos-mapping-service\PosMappingService.Data\tests\bin\Debug\PosMappingService.Data.Tests.exe
[2/14/2018 11:37:32 AM Informational] Creating test host for C:\Code\pos-mapping-service\PosMappingService.Web\tests\bin\Debug\PosMappingService.Web.Tests.exe
[2/14/2018 11:37:33 AM Informational] Executing tests in assembly C:\Code\pos-mapping-service\PosMappingService.Web\tests\bin\Debug\PosMappingService.Web.Tests.exe
[2/14/2018 11:37:33 AM Informational] Executing tests in assembly C:\Code\pos-mapping-service\PosMappingService.Data\tests\bin\Debug\PosMappingService.Data.Tests.exe
[2/14/2018 11:37:33 AM Error] System.MissingMethodException: Method not found: 'Boolean ExpectoConfig.get_allowDuplicateNames()'.
   at Execution.ExecuteProxy.ExecuteTests()
   at Execution.ExecuteProxy.ExecuteTests()
   at <StartupCode$Expecto-VisualStudio-TestAdapter>.$Execution.clo@180-10.Invoke(Unit unitVar)
   at Microsoft.FSharp.Control.AsyncBuilderImpl.callA@839.Invoke(AsyncParams`1 args)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.FSharp.Control.AsyncBuilderImpl.commit[a](AsyncImplResult`1 res)
   at Microsoft.FSharp.Control.CancellationTokenOps.RunSynchronouslyInCurrentThread[a](CancellationToken token, FSharpAsync`1 computation)
   at Microsoft.FSharp.Control.CancellationTokenOps.RunSynchronously[a](CancellationToken token, FSharpAsync`1 computation, FSharpOption`1 timeout)
   at Microsoft.FSharp.Control.FSharpAsync.RunSynchronously[T](FSharpAsync`1 computation, FSharpOption`1 timeout, FSharpOption`1 cancellationToken)
   at Execution.ExpectoTestExecutor.runAllExecutors()
   at Execution.ExpectoTestExecutor.Microsoft-VisualStudio-TestPlatform-ObjectModel-Adapter-ITestExecutor-RunTests(IEnumerable`1 sources, IRunContext runContext, IFrameworkHandle frameworkHandle)
[2/14/2018 11:37:33 AM Warning] No test is available in C:\Code\pos-mapping-service\PosMappingService.Data\tests\bin\Debug\PosMappingService.Data.Tests.exe C:\Code\pos-mapping-service\PosMappingService.Web\tests\bin\Debug\PosMappingService.Web.Tests.exe. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
[2/14/2018 11:37:33 AM Informational] ========== Run test finished: 0 run (0:00:01.5817122) ==========
```

**Comment by MNie** [comment]
@adam-c-anderson I think it is related to #36 and #22. Please try to use the newest version of an adapter.

**Comment by adam-c-anderson** [comment]
What's weird is that I've been using Expecto 5.0.1 and Expecto.VisualStudio.TestAdapter 6.0.0 successfully for a few weeks now until this. `paket update` didn't automatically update.

I'm still kind of new at paket, but I was able to get enough info out of it to suspect that the legacy pinned dependency on System.ValueTuple 4.3.1 might be preventing other things from updating. I removed the version constraint, re-ran `paket update` and gut up to Expecto 5.1.2 and Expecto.VisualStudio.TestAdapter 6.0.1. My tests now run in Visual Studio again. I still don't understand why the test adapter stopped working without any changes to either it or Expecto, but at this point I'm just happy to be off and running again!
